README.md

```diff
@@ -6,6 +6,82 @@ multibot
 
 A programmatic multi-repository helper.
 
+Multibot allows you to invoke actions (`read`, etc.) on 1+ files in 1+
+repositories concurrently. It is meant to be a helper tool for multi-project
+updates, branches, pull requests, etc.
+
+## Install
+
+```sh
+$ npm install multibot
+```
+
+## Usage
+
+```
+A friendly multi-repository robot.
+
+Usage: multibot --action=<string> [options]
+
+Options:
+  --action       Actions to take                               [string] [required] [choices: "read"]
```

Comment:
Is a future action going to be `"write"`, where multibot actually opens up a PR?

Is there a need for an action flag right now?
Should the default be `"read"`?